cmake/FindJavacc.cmake

```diff
@@ -8,7 +8,8 @@ if(JAVACC_EXECUTABLE)
       COMMAND "${JAVACC_EXECUTABLE}" --version
       OUTPUT_VARIABLE JAVACC_TEMP_VERSION
   )
-  string(REGEX MATCH ".* ([0-9]+\.[0-9]+[\.0-9]*) .*" JAVACC_TEMP_VERSION2_UNUSED "${JAVACC_TEMP_VERSION}")
+  string(REGEX MATCH ".* ([0-9]+\\.[0-9]+[\\.0-9]*) .*" JAVACC_TEMP_VERSION2_UNUSED "${JAVACC_TEMP_VERSION}")
```

Comment:
I'm sorry, I did a typo in the regex I suggested...
This even matches `7.0..........`.
`[\\.0-9]*` should be `(\\.[0-9])*`:
`".* ([0-9]+\\.[0-9]+(\\.[0-9])*) .*"`

Sorry for being nagging... :)